src/components/Table/DelegatesStickyTable.tsx

```diff
@@ -0,0 +1,81 @@
+import { useSortingByKey } from '../../hooks/useSortingByKey'
+import Sort from '../Icon/Sort'
+import DelegateRow from '../Table/DelegatesTableRow'
+
+import './DelegatesStickyTable.css'
+
+const DISPLAYED_CANDIDATES = 9
+const DEFAULT_SORTING_KEY: DelegateKey = 'lastVoted'
+type DelegateKey = keyof Delegate
+
+interface Props {
+  delegates: Delegate[]
+}
+
+const DelegatesStickyTable = ({ delegates }: Props) => {
+  const t = useFormatMessage()
+  const [sortingKey, setSortingKey] = useState(DEFAULT_SORTING_KEY as DelegateKey)
+  const { sorted: delegatesSorted, changeSort, isDescendingSort } = useSortingByKey(delegates, sortingKey)
+
+  const updateSort = (newSortingKey: DelegateKey) => {
+    if (newSortingKey !== sortingKey) {
+      setSortingKey(newSortingKey)
+    } else {
+      changeSort()
+    }
+  }
+
+  return (
+    <div className="DelegatesStickyTable__Wrapper">
+      <div className="DelegatesStickyTable__Scroller">
+        <Table unstackable basic="very" className="DelegatesStickyTable">
+          <Table.Header className="DelegatesStickyTable__Header">
+            <Table.Row>
+              <Table.HeaderCell className="DelegatesStickyTable__Fixed DelegatesStickyTable__CandidateName">
+                {t('page.home.dao_delegates.candidate_name')}
+              </Table.HeaderCell>
+              <Table.HeaderCell className="DelegatesStickyTable__ShadowBox DelegatesStickyTable__ShadowBoxHeader" />
+              <Table.HeaderCell onClick={() => updateSort('lastVoted')}>
+                <span>
+                  {t('page.home.dao_delegates.last_voted')}
+                  <Sort descending={isDescendingSort} selected={sortingKey === 'lastVoted'} />
+                </span>
+              </Table.HeaderCell>
+              <Table.HeaderCell onClick={() => updateSort('timesVoted')}>
+                <span>
+                  {t('page.home.dao_delegates.times_voted')}
+                  <Sort descending={isDescendingSort} selected={sortingKey === 'timesVoted'} />
+                </span>
+              </Table.HeaderCell>
+              <Table.HeaderCell onClick={() => updateSort('pickedBy')}>
+                <span>
+                  {t('page.home.dao_delegates.picked_by')}
+                  <Sort descending={isDescendingSort} selected={sortingKey === 'pickedBy'} />
+                </span>
+              </Table.HeaderCell>
+              <Table.HeaderCell onClick={() => updateSort('totalVP')}>
+                <span>
+                  {t('page.home.dao_delegates.total_vp')}
+                  <Sort descending={isDescendingSort} selected={sortingKey === 'totalVP'} />
+                </span>
+              </Table.HeaderCell>
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            {delegatesSorted.slice(0, DISPLAYED_CANDIDATES).map((delegate, index) => (
+              <DelegateRow key={delegate.address + 'row' + index} delegate={delegate} sticky />
```

Comment:
Isn't the index enough for the key? 🤔

Reply:
Not really :)

https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318